tasks/main.yml

```diff
@@ -1,14 +1,25 @@
 ---
 - name: Setup
-  sudo: yes
+  become: no
   file: dest={{ homebrew_root }} owner={{ ansible_ssh_user }} state=directory
 
 - name: Determine Homebrew version
   set_fact:
     homebrew_repo_name: "{{ 'homebrew' if ansible_os_family == 'Darwin' else 'linuxbrew' }}"
 
+- stat: path="~/{{ homebrew_repo_name }}"
+  register: linuxbrew_installed
+
+- name: Fetch
+  get_url:
+      url: https://raw.githubusercontent.com/Linuxbrew/install/master/install
```

Comment:
These changes are very Linuxbrew specific. Please make sure that everything works for both Linuxbrew and Homebrew (I suspect the latter has wider adoption than the former).